test/Compress/GzTest.php

```diff
@@ -177,4 +186,15 @@ public function testGzToString()
         $filter = new GzCompression();
         $this->assertEquals('Gz', $filter->toString());
     }
+
+    public function testGzDecompressNullThrowsRuntimeException()
+    {
+        $this->expectException(Exception\RuntimeException::class);
+        $this->expectExceptionMessage('Error during decompression');
+
+        $filter = new GzCompression();
+        $result = $filter->decompress(null);
+
+        $this->assertEmpty($result);
```

Comment:
shouldn't be needed as decompress method is throwing exception with null arg

Reply:
Fixed with b9e3ef4